playbook/app/pb_kits/playbook/pb_icon/docs/_icon_default.jsx

```diff
@@ -1,13 +1,29 @@
-import React from 'react'
+import React from "react"
 
-import Icon from '../_icon'
+import Icon from "../_icon"
+import {Caption} from "../.."
 
 const IconDefault = (props) => {
   return (
     <div>
-      <Icon
-          fixedWidth
-          icon="user"
+      <Icon fixedWidth
+          icon='angles-down'
+          {...props}
+      />
+      <Icon fixedWidth
+          icon='circle-arrow-right'
+          {...props}
+      />
+      <Icon fixedWidth
+          icon='arrow-circle-right'
+          {...props}
+      />
```

Comment:
These two icons show both the alias AND the icon directly from our playbook-icons working in conjunction with each other.